sql/moz-fx-data-shared-prod/udf/distribution_model_ga_metrics/udf.sql

```diff
@@ -0,0 +1,11 @@
+/*
+This is a stub implementation for use with tests in this repo
+Real implementation is in private-bigquery-etl
+*/
+CREATE OR REPLACE FUNCTION udf.distribution_model_ga_metrics()
+RETURNS STRING AS (
+  (SELECT 'helloworld' AS distribution_model)
+);
```

Comment:
Is there a difference between this and
```suggestion
RETURNS STRING AS (
  'helloworld'
);
```